FrontEndLib/Widget.h

```diff
@@ -442,6 +443,8 @@ class CWidget : public CAttachableObject
 private:
 	void        DrawRectClipped(const SDL_Rect &rect, const SURFACECOLOR &Color,
 			SDL_Surface *pDestSurface);
+
+	bool bIsPreventingBubbling; // Set to true in event handlers to prevent current event being called on parent
```

Comment:
Suggest rename: bPreventEventBubbling.
I'd recommend moving this comment to the Prevent(Event)Bubbling method (to clarify that's actually where the call is made to set this to true) and have something like the following comment here:
`// when true, don't pass received events on to parent object`